content/CORE/CORETutorials/Services/ConfigureUPS.md

```diff
@@ -38,4 +41,4 @@ The secondary computers are powered by the UPS, but receive UPS status data from
 See the [NUT User Manual](https://networkupstools.org/docs/user-manual.chunked/index.html) and [NUT User Manual Pages](https://networkupstools.org/docs/man/index.html#User_man).
 {{< /expand >}}
 
-See [UPS screen]({{< relref "/CORE/UIReference/Services/UPSScreen.md" >}}) for more information on setting up a UPS.
\ No newline at end of file
+{{< taglist tag="coreups" limit="10" title="Related UPS Articles" >}}
```

Comment:
Same comment as above regarding adding the title="Related xyz Articles" only when adding a second shortcode for a different tag. The "Related Articles" heading happens automatically with the basic shortcode without the title="".

Reply:
Changed description: "This article provides information on configuring UPS service on your TrueNAS."

Reply:
for some reason the title parameter still shows up in line 44?